.github/workflows/test_helper.yml

```diff
@@ -60,9 +60,8 @@ jobs:
 
           # this should probably be its own util/script
           export BUILD_ENGINE_DB=db-green-fast-track
-          cd products/green_fast_track
-          dbt seed
-          cd ../..
+          dbt deps --profiles-dir products/green_fast_track --project-dir products/green_fast_track
```

Comment:
Was it failing without this on your branch?

Also, I switched this from build to seed to avoid running seed tests while sqlfluffing. What functionality do you need from build?

Reply:
I think maybe I see why with the compile python reqs pr failing... is there only one test schema? I really didn't think about where this was actually running...

Reply:
> Also, I switched this from build to seed to avoid running seed tests while sqlfluffing. What functionality do you need from build?

testing the seed tables seems worth it. since we have to build them so that `sqlfluff ... --templater=dbt` works, seems worth validating those tables

I'm sure exactly why the tables have to exist while sqlfluffing (maybe `dbt compile` is run?) so maybe all that maters is that any seed table columns used in queries exist and the contents don't matter. but testing them seems to do more good than harm

Reply:
> is there only one test schema?

looks like there's only one for the job `check_formatting`, but the `dbt_checkpoint` uses a branch-unique schema. probably better to use the branch-unique schema for both

working on a fix in a different branch

Reply:
@fvankrieken if https://github.com/NYCPlanning/data-engineering/pull/842 is all good, I should drop this change from this branch